broker-daemon/orderbook/index.js

```diff
@@ -130,13 +144,56 @@ class Orderbook {
     })
   }
 
+  /**
+   * Gets current orderbook events by timestamp
+   *
+   * @param {String} timestamp - timestamp in nano-seconds
+   * @returns {Array<MarketEventOrder>}
+   */
+  async getOrderbookEventsByTimestamp (timestamp) {
+    this._assertSynced()
+    return getRecords(
+      this.store,
+      (key, value) => JSON.parse(value),
+      // Limits the query to gte to a specific timestamp
+      MarketEvent.rangeFromTimestamp(timestamp)
+    )
+  }
+
+  /**
+   * Gets MarketEvents by timestamp
+   *
+   * @param {String} timestamp - timestamp in nano-seconds
+   * @returns {Array<MarketEventOrder>}
+   */
+  async getMarketEventsByTimestamp (timestamp) {
+    this._assertSynced()
+    return getRecords(
+      this.eventStore,
+      (key, value) => JSON.parse(value),
+      // Limits the query to gte to a specific timestamp
+      MarketEvent.rangeFromTimestamp(timestamp)
+    )
+  }
+
+  /**
+   * Ensures that the orderbook is synced before accessing it
+   * @return {void}
+   * @throws {Error} If Orderbook is not synced to Relayer
+   */
+  _assertSynced () {
```

Comment:
[nit] Is this a convention we want to use going forward? I remember having a convo about this a few months ago and we decided to omit `underscores`. If we wanted a truly private method, we could add the function to the top of this file, and not export it.